### PR TITLE
form inputs added and logic implemented alongside modal

### PR DESCRIPTION
**Pull request by Moraheem98**
- Modal window has been added.
- form inputs and genre dropdown section has also been added.
- logic for for inputs have been added.
- form styling is yet to be implement, which will be done once all the logic is considered good.

### CODE REVIEW

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
The PR is not ready for review:
- linting errors present
- commented code present (entire components)

Additional notes:
- prop types error/warning might help you to distinguish components within other components - and components should be defined separately
- logic and markup should be separate (i.e avoid mapping directly in JSX)

File: task-2/src/components/AddMovieForm/index.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Dropdown } from '../Dropdown';
+import PropTypes from 'prop-types';
```
Comment (Dmitry-White) [inline task-2/src/components/AddMovieForm/index.js:4]: Consider grouping imports of the same level together (./oneThing and ../anotherThing are different levels) and separating each level with an empty line. Additionally, order imports top-down from most-global to most-local.
E.g. https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

If you've got time, consider adding this plugin to your project to automate that.

File: task-2/src/components/AddMovieForm/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+					placeholder='runtime'
+					name='runtimeName'
+					values={values.runtime}
+					onChange={handleInputChanges}
+				/>
+				<input
+					type='text'
+					className='inputField'
+					placeholder='description'
+					name='descriptionName'
```
Comment (Dmitry-White) [inline task-2/src/components/AddMovieForm/index.js]: Consider putting such change-sensitive and potentially repeating values into separate constants.
You can aggregate such values in an object with appropriate name outside of the component body.
E.g. `const ADD_FIELDS = { ..., DESCRIPTION_NAME = 'descriptionName', ...};`
And then here in `AddMovieForm` you'll have the following:
`<input ... name={ADD_FIELDS.DESCRIPTION_NAME} ... />`

File: task-2/src/components/Dropdown/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { dropdownOptions } from '../../core/data';
+
+export const Dropdown = () => {
+	const [dropdownValue, setDropdownValue] = useState('');
+
+	const dropdownHandler = (e) => {
+		setDropdownValue(e.target.value);
+	};
+
+	const handleDropdownValues = () => {
```
Comment (Dmitry-White) [inline task-2/src/components/Dropdown/index.js]: Consider changing the name to be closer to the function's purpose.
Here, the function essentially renders a list of components. So something like `renderOptions` would be more readable.

File: task-2/src/components/AddMovieForm/index.js (hunk 2 above)
Comment (Dmitry-White) [reply task-2/src/components/AddMovieForm/index.js]: Consider creating a "constants" folder under "src/core" where all the main reusable non-react things will be stored.

Comment (Dmitry-White) [reply task-2/src/components/AddMovieForm/index.js]: "constants" will hold any number of file to your liking - as long as they contain all the hard-coded numbers/strings that you use throughout the app. The idea is to have a central place for all the constants to be able to easily reuse/change them.

Comment (Dmitry-White) [reply task-2/src/components/AddMovieForm/index.js]: Once all the constants are in "src/core/constants" don't forget to go through the app and use them where applicable.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
Looks much better, thanks!

File: task-2/src/components/AddMovieForm/index.js (hunk 1 above)
Comment (Moraheem98) [reply task-2/src/components/AddMovieForm/index.js:4]: - Imports have been reorganised to match the correct convention.

File: task-2/src/components/AddMovieForm/index.js (hunk 2 above)
Comment (Moraheem98) [reply task-2/src/components/AddMovieForm/index.js]: - Constants folder has been created. 
- All reusable values have been moved into this folder.
- Form input "name" has been changed to match the dynamic nature of using constants.

File: task-2/src/components/Dropdown/index.js (hunk 3 above)
Comment (Moraheem98) [reply task-2/src/components/Dropdown/index.js]: - Dropdown function has been renamed to match the purpose of the function.

**Review by Dmitry-White — APPROVED** [review]
Looks awesome now!

**Pull request merged**